.github/workflows/on-push-to-main.yml

```diff
@@ -0,0 +1,192 @@
+name: On push to main
+
+on:
+  push:
+    branches: [main]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  SIMPLETEST_DB: "mysql://drupal:drupal@127.0.0.1:3306/drupal"
+  SIMPLETEST_BASE_URL: "http://127.0.0.1:8080"
+  MODULE_FOLDER: "drupal-cache"
+  MODULE_REPO: "momentohq/drupal-cache"
+  DRUPAL_MODULE_NAME: "momento_cache"
+  DRUPAL_CORE_VERSION: 9.4.x
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+  # MOMENTO_API_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
```

Comment:
Need to get this secret in place as well as `MOMENTO_MACHINE_USER_GITHUB_TOKEN`.

Reply:
these should both be done